debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -761,6 +817,21 @@ public static DdlParsingMode parse(String value, String defaultValue) {
                                                           .withDefault(true)
                                                           .withDescription("If set to true, we will only produce DML events into Kafka for transactions that were written on mysql servers with UUIDs matching the filters defined by the gtid.source.includes or gtid.source.excludes configuration options, if they are specified.");
 
+    /**
+     * If set to 'latest', connector when encountering new GTID channel after job restart will start reading it from the
+     * latest executed position (default). When set to 'earliest' the connector will start reading new GTID channels from the first available position.
+     * This is useful when in active-passive mysql setup during failover new GTID channel starts receiving writes, see #DBZ-923
+     *
+     * Defaults to latest.
+     */
+    public static final Field GTID_NEW_CHANNEL_POSITION = Field.create("gtid.new.channel.position")
```

Comment:
The new option also must be added to`configDef()` and `ALL_FIELDS`. I'll take care of it.